desktop/cmp/appbar/AppBar.js

```diff
@@ -47,6 +47,9 @@ export class AppBar extends Component {
          */
         hideAdminButton: PT.bool,
 
```

Comment:
Can we take advantage of this change to combine the admin button/feedback button/theme button/options button within a single framework menu of a single button?  Think its too many framework buttons that apps will rarely want to dedicate their prime real estate to.